### PR TITLE
chore: add layers management cypress tests

### PR DESCRIPTION
**Pull request by BRaimbault**
Adding tests to check:

the EarthEngineModal behavior works according to expectations,
adding and removing layers produces the expected effects.
Adding data-test properties when needed.

### CODE REVIEW

File: cypress/integration/manageLayers.cy.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
+import { EXTENDED_TIMEOUT, getApiBaseUrl } from '../support/util.js'
+
+const MAPS_ADMIN_AUTHORITY_ID = 'F_PROGRAM_TRACKED_ENTITY_ATTRIBUTE_GROUP_ADD'
```
Comment (jenniferarnesen) [inline cypress/integration/manageLayers.cy.js:3]: is this the correct authority?

File: cypress/integration/manageLayers.cy.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                (request) => {
+                    delete request.headers['if-none-match']
+                }
+            ).as('getDataStore')
+
+            cy.wait('@getDataStore', EXTENDED_TIMEOUT).then((interception) => {
+                cy.log(interception.response)
+            })
+
+            cy.request({
```
Comment (jenniferarnesen) [inline cypress/integration/manageLayers.cy.js:188]: Could you add some comments to each block in this test to explain what's happening?

File: cypress/integration/manageLayers.cy.js (hunk 1 above)
Comment (BRaimbault) [reply cypress/integration/manageLayers.cy.js:3]: chore: [da2968a](https://github.com/dhis2/maps-app/pull/3198/commits/da2968ad903b878ab66a55bdd6b8731bf6d69fc8)

File: cypress/integration/manageLayers.cy.js (hunk 2 above)
Comment (BRaimbault) [reply cypress/integration/manageLayers.cy.js:188]: chore: [da2968a](https://github.com/dhis2/maps-app/pull/3198/commits/da2968ad903b878ab66a55bdd6b8731bf6d69fc8)

**Pull request merged**